drf_spectacular/hooks.py

```diff
@@ -119,6 +119,14 @@ def create_enum_component(name, schema):
             enum_schema = {k: v for k, v in prop_schema.items() if k in ['type', 'enum']}
             prop_schema = {k: v for k, v in prop_schema.items() if k not in ['type', 'enum']}
 
+            # separate actual description from name-value tuples
+            if spectacular_settings.ENUM_GENERATE_CHOICE_DESCRIPTION:
+                if prop_schema.get('description', '').startswith('*'):
+                    enum_schema['description'] = prop_schema.pop('description')
+                elif '\n\n*' in prop_schema.get('description', ''):
+                    _, _, post = prop_schema['description'].partition('\n\n*')
```

Comment:
Apology for commenting on merging MR, @tfranzel .

Just a question about these changes...

We have a serializer with field (choices) and it has `help_text`. Unfortunately this help text is not preserved in the description of the enum due to this line. How we can preserve this description along with choices?

Reply:
@sshishov, the `help_text` should still be there in the serializer component (NOT the enum component as it belongs to the field). This could otherwise lead confusing descriptions actually belonging somewhere else.

If that is not the case, please open new issue with a reproduction and full example. thx